### ISSUE TITLE
Feature request: Aggiunta campi e rimozione funzionalità Stato Avanzamento

### ISSUE THREAD

**Issue by fredowashere** [SCAI-ITEC/SCAI-Modulo-attivita-frontend#19]
Le modifiche in oggetto riguardano il modulo di Stato Avanzamento.

1. Aggiunta campi filtri maschera di ricerca:
![per-marco-2](https://github.com/SCAI-ITEC/SCAI-Modulo-attivita-frontend/assets/117639221/d497c714-99c2-4dff-8546-d2adce598479)

2. Rimozione multiselezione e operazione multiriga in tabella dettaglio:
![per-marco-1](https://github.com/SCAI-ITEC/SCAI-Modulo-attivita-frontend/assets/117639221/de476b7b-e3f1-4133-add6-b4097742e7d3)

3. Aggiunta campi tabella di dettaglio:
To Be Decided @angelicazenere

**Comment by angelicazenere** [comment]
Come chiusa?

**Comment by marcoAgugiaro** [comment]
@angelicazenere mi sapresti indicare che campi vanno aggiunti alla tabella di dettaglio?
Così ti faccio la PR

**Comment by angelicazenere** [comment]
CostoCompetenza . Devo vedere se é dentro il rilascio 
Questo deve essere a livello commessa che edi dettaglio mensile

**Comment by angelicazenere** [comment]
Dei filtri é stato aggiunto mese?

**Comment by marcoAgugiaro** [comment]
No, non è mai stato menzionato il fatto di aggiungere mese ai filtri
